modules/oadp-configuring-node-agents.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.adoc
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-azure.adoc
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-gcp.adoc
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-mcg.adoc
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs.adoc
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-kubevirt.adoc
+// * virt/backup_restore/virt-backup-restore-overview.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-configuring-node-agents_{context}"]
+= Configuring node agents and node labels
+
+The DPA of {oadp-short} uses `nodeSelector` to select which nodes can run the node agent. `nodeSelector` is the simplest recommended form of node selection constraint.
+
+Any label specified must match the labels on each node.
+
+The correct way to run the node agent on any node you choose is for you to label the nodes with a custom label:
+
+[source,terminal]
+----
+'node-role.kubernetes.io/nodeAgent: ""'
```

Comment:
Maybe we can add a label command here for more clarification. 

$ oc label node<node-name> node-role.kubernetes.io/nodeAgent=""